pages/ambassadors/index.page.tsx

```diff
@@ -2,19 +2,19 @@ import React from 'react';
 import { getLayout } from '~/components/SiteLayout';
 import { SectionContext } from '~/context';
 
-import ambassadorsBanner from '../../public/img/community/ambassadors.png';
-import AmbassadorBanner from './components/AmbassadorBanner';
-import ambassadorData from '../../data/ambassadors.json';
-import AmbassadorCard from './components/AmbassadorCard';
-import ambassadorList from '../../data/ambassador_lists.json';
+import ambassadorData from '~/data/ambassadors.json';
```

Comment:
Importing this way is causing the workflow to fail due to TypeScript compiler type checking, the file should be imported using `fs` and `getStaticProps`.